### ISSUE TITLE
Result of projected image is different from tensorflow implementation

### ISSUE THREAD

**Issue by muhdlaziem** [NVlabs/stylegan2-ada-pytorch#57]
Dear NVlabs,

I tried projecting robert downey jr image using both tensorflow and pytorch version. The result is different. The tensorflow one is producing much better output. Im using Legacy stylegan2 weight.

![arobertdowneyjr_01 (1)](https://user-images.githubusercontent.com/33310853/109753863-ed2d5900-7c1d-11eb-9beb-24333c243195.png)
Original image

![arobertdowneyjr_01](https://user-images.githubusercontent.com/33310853/109753465-45b02680-7c1d-11eb-9103-aa82e58fc9f9.png)
Tensorflow projected image.

![proj](https://user-images.githubusercontent.com/33310853/109753495-55c80600-7c1d-11eb-8638-4f7fd9ec5803.png)
pytorch projected image

**Comment by MoemaMike** [comment]
yes very noticeable difference. OP any update on your end?

**Comment by tengshaofeng** [comment]
yes, I have the same question. Any body knows why? and how to fix it.
